modules/administration_manual/pages/appliance/Clamav.adoc

```diff
@@ -29,13 +29,13 @@ the checkboxes next to ``**clamav**'' (2), ``**clamav-freshclam**'', and
 After doing that, click ``**INSTALL**'' (3) above the listed packages,
 next to ``**SHOW DETAILS**'', to install them.
 
-image:appliance/ucs/clamav/install-clamav.png[image]
+image:appliance/ucs/clamav/install-clamav.png[UCS: Installing clamav, clamav-freshclam, and clamav-daemon.]
 
 After you do so, a confirmation dialog appears, as in the screenshot
 below, asking for confirmation to install the packages. Confirm the
 choice by again clicking ``**INSTALL**''.
 
-image:appliance/ucs/clamav/confirm-clamav-installation.png[image]
+image:appliance/ucs/clamav/confirm-clamav-installation.png[s2q]
```

Comment:
s2q?

Reply:
I did not qualify the content, this is the original from documentation.
Whatever this means, I migrated it.
I propose anyway to review a lot of content, but this would be a separate PR.

Reply:
As that's an alt tag, it'd make sense to make it more meaningful. I get, however, the script can't interpret these things on-the-fly.

Reply:
correct, I/we can change that to whatever we want to have written there, or even reset it to the default ``[image]``.
In case of changing, what text should be used?